docker/checkout.bash

```diff
@@ -46,5 +46,5 @@ else
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-msgs.git --branch develop
       git clone --depth=1 https://github.com/usdot-fhwa-stol/novatel_gps_driver.git --branch CARMANovatelGpsDriver_1.2.0
       git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-utils.git --branch develop
-      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch release/Wanderer
+      git clone --depth=1 https://github.com/usdot-fhwa-stol/carma-messenger.git --branch develop
```

Comment:
If we're going to specify a branch, it probably should be release/Wanderer, not develop, shouldn't it?